managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedgerConfig.java

```diff
@@ -411,7 +411,8 @@ public ManagedLedgerConfig setThrottleMarkDelete(double throttleMarkDelete) {
      *            time unit for retention time
      */
     public ManagedLedgerConfig setRetentionTime(int retentionTime, TimeUnit unit) {
-        this.retentionTimeMs = unit.toMillis(retentionTime);
+        // unit.toMillis(-1) = -60000
```

Comment:
This comment is confusing. What do you mean?

Reply:
```
System.out.println("time -1 = " + TimeUnit.MINUTES.toMillis(-1));

time -1 = -60000
```

Reply:
`unit` is not always `TimeUnit.MINUTES`.

Anyway, you removed this change so it does not make any difference now.